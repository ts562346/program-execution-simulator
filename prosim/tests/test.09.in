test1 6
BLOCK 1
LOOP 4
  DOOP 2
END
BLOCK 1
HALT

