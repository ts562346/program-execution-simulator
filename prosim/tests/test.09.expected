00000: BLOCK
00001: DOOP
00003: DOOP
00005: DOOP
00007: DOOP
00009: BLOCK
00010: HALT

DOOP count : 4
DOOP time  : 8
BLOCK count: 2
BLOCK time : 2
